frontend/packages/vkt/src/components/publicEnrollment/PublicEnrollmentStepper.tsx

```diff
@@ -24,11 +27,17 @@ export const PublicEnrollmentStepper = ({
     }
   };
 
-  const getDescription = (stepNumber: number) =>
-    t(`step.${PublicEnrollmentFormStep[stepNumber]}`);
+  const getDescription = (stepNumber: number) => {
+    const i =
+      !includePaymentStep && stepNumber >= PublicEnrollmentFormStep.Pay
```

Comment:
Syy miksi täällä verrataan Pay-steppiin ja yllä Done-steppiin liittyy sellaiseen, että näin voitaisiin kai teoriassa lisätä myös jotain steppejä näiden kahden väliin jos jostain syystä olis tarvis